tests/plugins/emoji/manual/emoji.md

```diff
@@ -0,0 +1,27 @@
+@bender-tags: 4.10.0, feature, emoji, 1746
+@bender-ckeditor-plugins: wysiwygarea, toolbar, elementspath, sourcearea, emoji, clipboard, undo, stylescombo, format
+@bender-ui: collapsed
+
+# Test case #1:
+1. Inserting new emoji by typing it in editor.
```

Comment:
This is not clear enough. One could ask:

> By typing what?

Imagine you're seeing this test for the first time ever.